validate/validators/cram.py

```diff
@@ -0,0 +1,48 @@
+'''Helper methods for CRAM file validation'''
+from typing import Dict, Union
+from pathlib import Path
+from collections import namedtuple
+
+import pysam
+
+CramArgs = namedtuple(
+    'args',
+    'cram_reference'
+)
+
+def validate_cram_file(path, reference=None):
```

Comment:
Can you add type hints for `path` and `reference`? In tracing the code from `validate_file` to here, it is challenging to keep in mind the types coming into this function, as well as what the other `validate*` and `check*` functions need.